### PR TITLE
Implement Hello class

### PR DESCRIPTION
**Pull request by DoctorRoot**
@DoctorRoot, I don't know why my code isn't working, can you please take a look.

### CODE REVIEW

File: main.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class Hello
+
+  def say_hello
+    puts "hello world
```
Comment (shedeed) [inline main.rb]: Missing a "

Comment (nasseef) [inline main.rb]: missed a quote.

**Comment by professorkarlacarter** [comment]
Remember to have your quotation marks match :-)

File: main.rb (hunk 1 above)
Comment (abukmail) [inline main.rb]: You are missing a quote. Add a " at the end of the line.

Comment (ghost) [inline main.rb]: Missing closing quote

Comment (kendrickhang) [inline main.rb]: Looks like you are missing the closing double quote here to end the string literal ( "hello world" )

**Pull request merged**